webhooks-extension/README.md

```diff
@@ -1,72 +1,62 @@
-```
+*If running on Docker for Desktop*
 
-## API Definitions
+- Knative requires a Kubernetes cluster running version v.1.11 or greater. Currently this requires the edge version of Docker for Desktop. Your cluster must also be supplied with sufficient resources _(6 CPUs, 10GiB Memory & 2.5GiB swap is known good)_.
 
-- [Extension API definitions](cmd/extension/README.md)
-  - The extension API endpoints can be accessed through the dashboard.
-- [Sink API definitions](cmd/sink/README.md)
+## Install Quickstart
 
-### Example creating a webhook
+### Domain setup for Knative Serving
 
-You should be able to use the extension via the Tekton Dashboard UI - however, until the UI is coded or if you would prefer to interact with REST endpoint directly, you can create your webhook using curl:
+Set your own domain and selectors following the [configuring Knative Serving docs](https://github.com/knative/serving/blob/master/install/CONFIG.md) which outline setting up routes in the `config-domain` ConfigMap in the `knative-serving` namespace
 
-```bash
-data='{
-  "name": "go-hello-world",
-  "namespace": "green",
-  "gitrepositoryurl": "https://github.com/ncskier/go-hello-world",
-  "accesstoken": "github-secret",
-  "pipeline": "simple-pipeline",
-  "dockerregistry": "mydockerregistry"
-}'
-curl -d "${data}" -H "Content-Type: application/json" -X POST http://localhost:8080/webhooks
-```
+On Docker Desktop, you can retrieve your IP and patch it to the ConfigMap by running:
 
-When curling through the dashboard, use the same endpoints; for example, assuming the dashboard is at `localhost:9097`:
+`ip=$(ifconfig | grep netmask | sed -n 2p | cut -d ' ' -f2)`
 
-```bash
-curl -d "${data}" -H "Content-Type: application/json" -X POST http://localhost:9097/webhooks
+```
+kubectl patch configmap config-domain --namespace knative-serving --type='json' \
+  --patch '[{"op": "add", "path": "/data/'"${ip}.nip.io"'", "value": ""}]'
 ```
 
-Reference the [Knative eventing GitHub source sample](https://knative.dev/docs/eventing/samples/github-source/) to properly create the `accesstoken` secret. This is the secret that is used to create GitHub webhooks.
+### Install the Webhooks Extension 
 
-## Limitations
+The Tekton Webhooks Extension has hosted images located at `gcr.io/tekton-nightly/extension:latest` and `gcr.io/tekton-nightly/extension:latest`, to install the latest extension and sink using these images:
 
-- Only Git Hub webhooks are currently supported.
-- Only `push` and `pull_request` events are supported at the moment. The webhook is currently only created with both `push` and `pull_request` events.
-- All knative event sources are created in the namespace into which the dashboard and this extension are installed.
-- Only one webhook can be created for each git repository, so each repository will only be able to trigger a PipelineRun from one webhook.
+`kubectl apply -f config/release/gcr-tekton-webhooks-extension.yaml`
```

Comment:
It's all installed, now what? Do I port-forward, proxy? Go to an ingress endpoint? NodePort?